### ISSUE TITLE
Suggestion: add ability to call the lawyer

### ISSUE THREAD

**Issue by billfreeman44** [scrudgey/yogurt#141]
Maybe he can only be called from the corporate office.  Might be a fun addition

**Comment by scrudgey** [comment]
good idea- looking for more phone numbers to include, they may be unlockable

**Comment by billfreeman44** [comment]
Perhaps for the scorpion gang commercial, you have to call them to get them to come so that people are introduced to the phone call mechanic
